process-engine/consumer_api_contracts
# 🔥 Disallow non-JSON payloads for ExternalTask results

## Changes

Impose type restrictions on `ExternalTaskSuccessResult` to ensure that only valid JSONs or serializable object types are used as result.

## Issues

Part of https://github.com/process-engine/process_engine_runtime/issues/503

PR: #90

## How to test the changes

- Try creating an `ExternalTaskSuccessResult` with a string, number or boolean payload
- Get an error
- Try creating an `ExternalTaskSuccessResult` with a JSON result or any kind of instantiable object
- See that it works
